<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camLightDlg</class>
 <widget class="QWidget" name="camLightDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>389</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget:pane {background:transparent; }</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>光源设置</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1" columnminimumwidth="1,2,1">
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>串口编号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_LightPort">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>背光：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_LightValue">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_LightValue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>红光：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_LightValue1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_LightValue1">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>绿光：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="horizontalSlider_LightValue2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spinBox_LightValue2">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>蓝光：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="horizontalSlider_LightValue3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="spinBox_LightValue3">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>206</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>相机设置</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>基本信息：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>厂商：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="cameravendor">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>型号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="cameramodel">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>序列号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="cameraserial">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>触发模式：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="singlemodel">
          <property name="text">
           <string>单帧</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="continuousmodel">
          <property name="text">
           <string>连续</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="externalmodel">
          <property name="text">
           <string>外触发</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>参数设置：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>曝光：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="expouseslider">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="expousebox">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>22</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>增益：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="gainslider">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="gainbox">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>22</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>延时：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="spcdslider">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="spcdbox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>22</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>图像参数：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>图像最大宽度：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="image_maxwidth">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>图像最大高度：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="image_maxheight">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ROI区域宽度：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="image_width">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>ROI区域高度：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="image_height">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>ROI水平偏移：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="offsetX">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>ROI垂直偏移：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="offsetY">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>水平Binning：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="binningX">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>垂直Binning：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="binningY">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>相机防抖时间：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineTime">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="imagehori">
          <property name="text">
           <string>水平镜像</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="imageveri">
          <property name="text">
           <string>垂直镜像</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_rotateImage">
          <property name="text">
           <string>是否启用图像旋转</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>LSC分块宽度：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_secNumW"/>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>LSC分块高度：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="lineEdit_secNumH"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>LSC校正系数：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_padcoef"/>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>LSC标定方式：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="lineEdit_calibmethod"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>LSC目标灰度：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_targetgray"/>
        </item>
        <item row="8" column="2" colspan="2">
         <widget class="QCheckBox" name="checkBox_isNeedLSCCorrect">
          <property name="text">
           <string>是否启用阴影校正</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>7</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
